hazelcast-spring/src/test/java/com/hazelcast/spring/cache/JCacheCacheManagerTest.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.spring.cache;
+
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.spring.CustomSpringJUnit4ClassRunner;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.cache.jcache.JCacheCacheManager;
+import org.springframework.test.context.ContextConfiguration;
+
+import javax.annotation.Resource;
+import javax.cache.CacheManager;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+@RunWith(CustomSpringJUnit4ClassRunner.class)
+@ContextConfiguration(locations = {"jCacheCacheManager-applicationContext-hazelcast.xml"})
+@Category(QuickTest.class)
+public class JCacheCacheManagerTest {
+
+    @Resource(name = "instance")
+    private HazelcastInstance instance;
+
+    @Autowired
+    private IJCacheDummyBean bean;
+
+    @Resource(name = "cacheManager")
+    private JCacheCacheManager springCacheManager;
+
+    @Resource(name = "cacheManager2")
+    private CacheManager cacheManager2;
+
+    @BeforeClass
+    @AfterClass
+    public static void start() {
+        Hazelcast.shutdownAll();
```

Comment:
I still do not understand this one. Why is it needed? Hazelcast instance is started by the container, shouldn't the container stop it on its destruction?